src/main/resources/com/mathworks/ci/RunMatlabCommandStep/config.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    
+	  <f:entry title="Command" field="matlabCommand">
```

Comment:
I thought we were going with `command` here instead of `matlabCommand`?

Reply:
in script we need not enter this parameter at all as you see the snippet . However i have changed it to command to make it consistent